internal/pkg/deploy/task.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package deploy holds the structures to deploy infrastructure resources.
+// This file defines service deployment resources.
+package deploy
+
+type CreateTaskResourcesInput struct {
```

Comment:
Can we add a comment here? Go best practices indicate we should add comments to all exported structs, consts, functions, and methods.